tests/functional/tests.d/341-selfaccesses-force-password.sh

```diff
@@ -0,0 +1,239 @@
+            success g1_create $a0 --osh groupCreate --group $group1 --owner $account0 --no-key
+            json .error_code OK .command groupCreate
+            revoke groupCreate
+
+            # add account1 as member
+            success g1_member_a1 $a0 --osh groupAddMember --group $group1 --account $account1
+            json .error_code OK .command groupAddMember
+        fi
+
+        # missing hash
+        run ${mode}_add_a4_fp_nohash $a0 --osh $add_access_plugin $target --host $remote_ip --user $account4 --port $remote_port --force-password
+        retvalshouldbe 100
+        contain "Option force-password requires an argument"
+        json .error_code ERR_BAD_OPTIONS .command $add_access_plugin
+
+        # invalid hash
+        run ${mode}_add_a4_fp_invalidhash $a0 --osh $add_access_plugin $target --host $remote_ip --user $account4 --port $remote_port --force-password "invalid"
+        retvalshouldbe 100
+        contain "Specified hash is invalid"
+        json .error_code ERR_INVALID_PARAMETER .command $add_access_plugin
+
+        # test if the forced password appears in the access list
+        fake_hash='$5$fakefake$fakefakefakefakefakefakefakefakefakefakefak'
+        success ${mode}_add_a4_fp_fake $a0 --osh $add_access_plugin $target --host $remote_ip --user $account4 --port $remote_port --force-password "'${fake_hash}'"
+        json .error_code OK .command $add_access_plugin
+
+        grant accountListAccesses
+        success ${mode}_listaccess $a0 --osh accountListAccesses --account $account1
+        contain "FORCED-PASSWORD"
+        json .error_code OK .command accountListAccesses .value[0].acl[0].forcePassword $fake_hash
+        revoke accountListAccesses
+
+        success ${mode}_del_a4_fp_fake $a0 --osh $del_access_plugin $target --host $remote_ip --user $account4 --port $remote_port
+        json .error_code OK .command $del_access_plugin
+
+        # add a few egress passwords to account1|group1
+        success ${mode}_gen_pass1 $a0 --osh $gen_pass_plugin $target --do-it
+        json .error_code OK .command $gen_pass_plugin
+        success ${mode}_gen_pass2 $a0 --osh $gen_pass_plugin $target --do-it
+        json .error_code OK .command $gen_pass_plugin
+        success ${mode}_gen_pass3 $a0 --osh $gen_pass_plugin $target --do-it
+        json .error_code OK .command $gen_pass_plugin
+        success ${mode}_gen_pass4 $a0 --osh $gen_pass_plugin $target --do-it
+        json .error_code OK .command $gen_pass_plugin
+
+        # overwrite a1|g1's second egress password with account4's ingress password
+        success ${mode}_overwrite_pass2 $r0 "\"echo ${a4_password} > ${password_base_path}.1\""
+
+        # fetch checksums for a1|g1's second and third egress passwords
+        success ${mode}_listpass $a0 --osh $list_pass_plugin $target
+        json .error_code OK .command $list_pass_plugin
+        password2_sha256=$(get_json |  jq '.value[1].hashes.sha256crypt' | sed -e 's/"//g')
```

Comment:
you can just use `jq -r` here, to get the raw output, instead of sed'ing out the `"`

Reply:
Done